### ISSUE TITLE
Page title color

### ISSUE THREAD

**Issue by ksheedlo** [rackerlabs/canon-bootstrap#61]
Page titles in Canon are red and constructed with an `h2.rs-page-title`, see http://rackerlabs.github.io/canon/ui-components/#typography.

How can we copy this look and feel in canon-bootstrap?

**Comment by bradgignac** [comment]
You could use [contextual color classes](http://getbootstrap.com/css/#helper-classes-colors) for the time being, but it isn't exactly semantic.

**Comment by ksheedlo** [comment]
I can do whatever I want at the application layer (for now I made a `.u-pageTitle` class that sets the color) but this is a problem that I feel should be solved by the framework.

**Comment by bradgignac** [comment]
Agreed. I'll leave this issue open until someone has time to submit a PR.
